### DISCUSSION TITLE
Healthcheck endpoint

### DISCUSSION THREAD

**Discussion by stavros-k** [kimai/kimai#3222] [General]
I've looked around, but I can't seem to find a healthcheck endpoint.
The only thing that might be close to it is the `/api/ping` but that needs authentication. 

I'm deploying in kubernetes and I'd like to setup probes for it.

Is such endpoint available? That would return 200 only if the server is ready/ok?

**Comment by kevinpapst** [comment]
I think there is nothing available. 
You can curl the login page, but I guess it will render even if the database is down...

**Reply by stavros-k** [reply]
That's what I'm doing now. 
But such an endpoint would be a nice feature to be added
